### PR TITLE
Mgmt 4801 - Use common GetHostFromDB in the server

### PR DESCRIPTION
**Pull request by michaellevy101**

### CODE REVIEW

File: internal/bminventory/inventory.go [hunk 1]
```diff
@@ -2542,21 +2526,17 @@ func (b *bareMetalInventory) DeregisterHost(ctx context.Context, params installe
...
-	var host models.Host
+func (b *bareMetalInventory) GetHost(_ context.Context, params installer.GetHostParams) middleware.Responder {
 	// TODO: validate what is the error
-	if err := b.db.Where("id = ? and cluster_id = ?", params.HostID, params.ClusterID).
-		First(&host).Error; err != nil {
-		return installer.NewGetHostNotFound().WithPayload(common.GenerateError(http.StatusNotFound, err))
-	}
+	host, err := common.GetHostFromDB(b.db, params.ClusterID.String(), params.HostID.String())
 
-	if err := b.customizeHost(&host); err != nil {
```
Comment (YuviGold) [inline internal/bminventory/inventory.go:2553]: Where `customizeHost` disappeared?

Comment (michaellevy101) [reply internal/bminventory/inventory.go:2553]: It's a cosmetic change.... I change the host to be a pointer instead of object and remove the '&'....

Comment (michaellevy101) [reply internal/bminventory/inventory.go:2553]: will fix. probably a mistake

**Review by YuviGold — APPROVED** [review]
LGTM

**Pull request merged**